Math/src/main/java/org/tribuo/math/util/VectorNormalizer.java

```diff
@@ -50,21 +47,17 @@ default public void normalizeInPlace(double[] input) {
     }
 
     /**
-     * Exports this normalizer as a list of ONNX NodeProtos.
+     * Exports this normalizer to ONNX, returning the leaf of the appended graph.
```

Comment:
I think this should be a little more explicit (like EnsembleCombiner and ONNXExportable are) that it writes nodes into the ONNX context.